shell/agents/Microsoft.Azure.Agent/Command.cs

```diff
@@ -53,7 +53,7 @@ private void ReplaceAction()
 
         if (ap is null)
         {
-            host.WriteErrorLine("No argument placeholder to replace.");
+            host.WriteErrorLine("The '/replace' command is experimental and could not successfully parse the response. This issue may occur intermittently due to specific response conditions.");
```

Comment:
https://github.com/PowerShell/ProjectMercury/pull/289 was merged, so this change can be reverted.